MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -24,6 +24,28 @@
 
 import Foundation
 
+/**
+    Subclasses MSALPublicClientApplication to be used for Native Auth operations
```

Comment:
Why should we specify that MSALNativeAuthPublicClientApplication is a subclass of MSALPublicClientApplication? I think we shouldn't say that, it's an internal technical detail

Reply:
This is based on the discussion with Rodhan but indeed looking at Android one, we should have something more aligned so I changed it - https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/nativeauth/INativeAuthPublicClientApplication.kt